Dockerfile

```diff
@@ -1,62 +1,48 @@
 #
 # Dockerfile for Verificarlo (github.com/verificarlo/verificarlo)
-# This image includes support for Fortran and uses llvm-3.6.1 and gcc-4.9.
+# This image includes support for Fortran and uses llvm-7 and gcc-7
 #
 
-FROM ubuntu:16.04
+FROM ubuntu:20.04
 MAINTAINER Verificarlo contributors <verificarlo@googlegroups.com>
 
-ARG PYTHON_VERSION=3.5
-ARG LLVM_VERSION=3.6.1
-ARG GCC_VERSION=4.9
+ARG PYTHON_VERSION=3.8
+ARG LLVM_VERSION=7
+ARG GCC_VERSION=7
 ARG GCC_PATH=/usr/lib/gcc/x86_64-linux-gnu/${GCC_VERSION}
 ENV LD_LIBRARY_PATH /usr/local/lib:$LD_LIBRARY_PATH
 ENV PATH /usr/local/bin:$PATH
 ENV PYTHONPATH=/usr/local/lib/python$PYTHON_VERSION/site-packages/:$PYTHONPATH
 
 # Retrieve dependencies
-RUN apt-get -y update && apt-get -y install --no-install-recommends \
+RUN apt-get -y update && apt-get -y --no-install-recommends install tzdata 
+RUN apt-get -y install --no-install-recommends \
     bash ca-certificates make git libmpfr-dev \
-    gcc-${GCC_VERSION} gcc-${GCC_VERSION}-plugin-dev g++-${GCC_VERSION} gfortran-${GCC_VERSION} libgfortran-${GCC_VERSION}-dev \
-    autoconf automake libedit-dev libtool libz-dev \
-    python3 python3-dev python3-numpy python3-matplotlib python3-pip python3-setuptools binutils vim sudo wget xz-utils && \
-    rm -rf /var/lib/apt/lists/
+    autogen dh-autoreconf autoconf automake autotools-dev libedit-dev libtool libz-dev binutils \
+    clang-${LLVM_VERSION} llvm-${LLVM_VERSION} llvm-${LLVM_VERSION}-dev \
+    gcc-${GCC_VERSION} g++-${GCC_VERSION} \
```

Comment:
Same question than before: can we remove gcc and g++ dependencies completely ?

Reply:
Same answer as before.